src/docs/faq.md

```diff
@@ -7,105 +7,134 @@ Please read this page before asking for help. Your help request will be pointed
-* Windows 11 is required for TB4 users. If you don't have Thunderbolt 4, you may use both 10 or 11.
+
+- Windows 11 is required for TB4 users. If you Do not have Thunderbolt 4, you may use both 10 or 11.
 
 ### Can I pay for Windows drivers using PayPal?
-* You can pay via PayPal in Patreon.
+
+- You can pay via PayPal in Patreon.
 
 ### My balance shows zero on coolstars portal, what should I do?
-* It may take up to 48 hours for balance to appear in her portal, though it is normally less then 2.
+
+- It may take up to 48 hours for balance to appear in her portal, though it is normally less then 2.
 
 ### How do I buy the Windows audio/Thunderbolt drivers?
-* Please see [CoolStar's driver portal](https://coolstar.org/chromebook/driverlicense/login.html) for a step by step tutorial on purchasing drivers.
+
+- Please see [CoolStar's driver portal](https://coolstar.org/chromebook/driverlicense/login.html) for a step by step tutorial on purchasing drivers.
 
 ### Can I get AVS or SOF drivers for free?
-* No. Only the Linux drivers are free.
+
+- No. Only the Linux drivers are free.
 
 ### Why is there a license error in Device Manager after installing audio drivers?
-* You messed up the license setup. Please refer to [CoolStar's driver portal](https://coolstar.org/chromebook/driverlicense/login.html).
+
+- You messed up the license setup. Please refer to [CoolStar's driver portal](https://coolstar.org/chromebook/driverlicense/login.html).
 
 ### Will the audio driver still work when re-installing Windows?
-* Yes, on the device the license was created. If the internal drive was replaced, see below.
+
+- Yes, on the device the license was created. If the internal drive was replaced, see below.
 
 ### I upgraded my Chromebooks' internal drive and my audio/Thunderbolt 4 license is not working, what do I do?
-* The license is tied to the serial number of your internal drive. **You do not need to buy another license**. Send CoolStar a message through Patreon, and wait for about a week.
+
+- The license is tied to the serial number of your internal drive. **You do not need to buy another license**. Send CoolStar a message through Patreon, and wait for about a week.
 
 ### Can I use one driver license across multiple devices?
-* No. One license per device.
+
+- No. One license per device.
 
 <br>
 
 ## Linux Questions
 
 ### How come audio isn't working on Ubuntu / Ubuntu forks?
-* Ubuntu and Ubuntu-based distros may have issues and are not supported, try switching to another distro. Debian is a great alternative if you are used to Ubuntu.
+
+- Ubuntu and Ubuntu-based distros may have issues and are not supported, try switching to another distro. Debian is a great alternative if you are used to Ubuntu.
 
 ### Why are Ubuntu/Ubuntu based distros not supported?
-* They consistently break packages.
-* They have started to force snap on their users.
-* They have published an LTS distro with a non-LTS kernel.
+
+- They consistently break packages.
```

Comment:
(Not really a recommendation just an ubuntu hate comment)

That and their operating system live iso is literally as large as windows. Like go use windows at that point, youre getting the same privacy and storage usage.